feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle-dropdown/configurator-attribute-single-selection-bundle-dropdown.component.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Configurator } from '../../../../core/model/configurator.model';
+import { ConfiguratorAttributeProductCardComponentOptions } from '../../product-card/configurator-attribute-product-card.component';
+import { ConfiguratorAttributeSingleSelectionBaseComponent } from '../base/configurator-attribute-single-selection-base.component';
+
+@Component({
+  selector: 'cx-configurator-attribute-single-selection-bundle-dropdown',
+  templateUrl:
+    './configurator-attribute-single-selection-bundle-dropdown.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfiguratorAttributeSingleSelectionBundleDropdownComponent
+  extends ConfiguratorAttributeSingleSelectionBaseComponent
+  implements OnInit {
+  attributeDropDownForm = new FormControl('');
+  selectionValue: Configurator.Value | undefined;
```

Comment:
it is better `selectionValue?: Configurator.Value;`

Reply:
Picked!

Reply:
Done!